src/minidump.rs

```diff
@@ -2007,13 +2284,76 @@ impl fmt::Display for CrashReason {
     /// "EXC_BAD_ACCESS / KERN_INVALID_ADDRESS" (Mac OS X), "SIGSEGV"
     /// (other Unix).
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(
-            f,
-            "{}",
-            match *self {
-                CrashReason::Unknown => "unknown",
+        use CrashReason::*;
+
+        fn write_nt_status(_f: &mut fmt::Formatter<'_>, _nt_status: u64) -> fmt::Result {
+            // TODO: implement this
+            Ok(())
+        }
+
+        fn write_fast_fail(_f: &mut fmt::Formatter<'_>, _fast_fail: u64) -> fmt::Result {
+            // TODO: implement this
+            Ok(())
+        }
+
+        // OK this is kinda a gross hack but I *really* don't want
+        // to write out all these strings again, so let's just lean on Debug
+        // repeating the name of the enum variant!
```

Comment:
This seems fine. If you want to pull in a proc macro crate that offers `#[derive(Display)]` for enums at some point, feel free.